MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "com_github_yohhoy_yamc",
+    version = "1.0.0",
```

Comment:
The `name` here is how this repository would show up to developers. 

I don't know what `version` you would prefer here since I don't see a cut release in the repository. Let me know!

If you have any preferences please let me know or feel free to edit as you wish!